packages/styles/src/types.ts

```diff
@@ -188,6 +188,8 @@ export interface ComponentSlotStylesInput<TProps = {}, TVars = {}>
 export interface ComponentSlotStylesPrepared<TProps = {}, TVars = {}>
   extends Record<string, ComponentSlotStyleFunction<TProps, TVars>> {}
 
+export interface ComponentSlotStylesResolved extends Record<string, ICSSInJSStyle> {}
```

Comment:
Can we please extract this change to a separate PR? It's absolutely valid, but it creates noise in this PR